SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPropertyReplication/SpatialTestPropertyReplication.cpp

```diff
@@ -24,32 +24,37 @@ ASpatialTestPropertyReplication::ASpatialTestPropertyReplication()
 	: Super()
 {
 	Author = "Ollie Balaam (oliverbalaam@improbable.io)";
-	Description = TEXT("This tests that an Actor can replicate a property across the network during play. It is an example test intended to teach the basics of the UnrealGDK Functional Test Framework. It's accompanied by this document: https://brevi.link/how-to-test-unrealgdk");
+	Description = TEXT(
+		"This tests that an Actor can replicate a property across the network during play. It is an example test intended to teach the "
+		"basics of the UnrealGDK Functional Test Framework. It's accompanied by this document: https://brevi.link/how-to-test-unrealgdk");
 }
 
 void ASpatialTestPropertyReplication::PrepareTest()
 {
 	Super::PrepareTest();
 
-AddStep(
+	AddStep(
 		TEXT("Check PIE override settings"), FWorkerDefinition::AllServers, nullptr,
 		[this]() {
 			int32 ExpectedNumberOfClients = 3;
 			int32 RequiredNumberOfClients = GetNumRequiredClients();
-		RequireEqual_Int(RequiredNumberOfClients, ExpectedNumberOfClients, TEXT("Expected a certain number of clients to be required."));
-		int32 ActualNumberOfClients = GetNumberOfClientWorkers();
-		RequireEqual_Int(ActualNumberOfClients , ExpectedNumberOfClients, TEXT("Expected a certain number of clients to actually connect."));
-		FinishStep();
+			RequireEqual_Int(RequiredNumberOfClients, ExpectedNumberOfClients,
+							 TEXT("Expected a certain number of clients to be required."));
+			int32 ActualNumberOfClients = GetNumberOfClientWorkers();
+			RequireEqual_Int(ActualNumberOfClients, ExpectedNumberOfClients,
+							 TEXT("Expected a certain number of clients to actually connect."));
+			FinishStep();
 		},
 		nullptr, 5.0f);
 
-	AddStep(TEXT("The Server spawns one ReplicatedTestActor"), FWorkerDefinition::Server(1), nullptr, [this]() {
-		TestActor =
-			GetWorld()->SpawnActor<AReplicatedTestActor>(FVector(0.0f, 0.0f, 50.0f), FRotator::ZeroRotator, FActorSpawnParameters());
-		RegisterAutoDestroyActor(TestActor);
+	AddStep(
+		TEXT("The Server spawns one ReplicatedTestActor"), FWorkerDefinition::Server(1), nullptr,
+		[this]() {
+			TestActor = SpawnActor<AReplicatedTestActor>(FVector(0.0f, 0.0f, 50.0f));
```

Comment:
we should update https://brevi.link/how-to-test-unrealgdk to reflect this new recommended usage, mention that it registers the actor for destruction internally

Reply:
updated